assemblies/assembly_openshift-operator/main.adoc

```diff
@@ -14,13 +14,13 @@ As a system engineer, you can deploy a {PRODUCT} environment on {OPENSHIFT} to p
 
 .Prerequisites
 
-* A {OPENSHIFT} environment is available. The Operator is supported on {OPENSHIFT} version 4.1 and 4.2.
+* A {OPENSHIFT} environment is available. The Operator is supported on {OPENSHIFT} version 4.1, 4.2, and 4.3.
 * At least four gigabytes of memory are available in the OpenShift environment.
 * The OpenShift project for the deployment is created.
 * You are logged in to the project using the OpenShift web console.
 * Dynamic persistent volume (PV) provisioning is enabled. Alternatively, if dynamic PV provisioning is not enabled,  enough persistent volumes must be available. By default, the deployed components require the following PV sizes:
 ifdef::PAM[]
-** Each deployed replicated set of {KIE_SERVER} pods, by default, requires one 1Gi PV for the database. You can change the database PV size. You can deploy multiple immutable servers; each requires a separate database PV. This requirement does not apply if you use an external database server.
+** Each {KIE_SERVER} deployment by default requires one 1Gi PV for the database. You can change the database PV size. You can deploy multiple {KIE_SERVERS}; each requires a separate database PV. This requirement does not apply if you use an external database server.
```

Comment:
I would remove "by default". Does not seem necessary.

** Each {KIE_SERVER} deployment requires one 1Gi PV for the database. You can change the database PV size. You can deploy multiple {KIE_SERVERS}; each requires a separate database PV. This requirement does not apply if you use an external database server.